Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.py

```diff
@@ -0,0 +1,1078 @@
+    :rtype: ``CommandResults``
+    """
+
+    domain = str(args.get('domain'))
+    _from = args.get('from')
+    to = args.get('to')
+    timing = str(args.get('timing'))
+
+    demisto.debug("Arguments: {0}".format(args))
+    response = client.get_company_historical_factor_scores(domain=domain, _from=_from, to=to, timing=timing)  # type: ignore
+
+    demisto.debug("API response: {0}".format(response))
+
+    entries = response['entries']
+
+    factor_scores = []
+
+    for entry in entries:
+        f = {}
+        f["Date"] = entry.get("date").split("T")[0]
+
+        factors = entry.get("factors")
+        factor_row = ''
+        for factor in factors:
+            factor_name = factor.get("name").title().replace("_", " ")
+            factor_score = factor.get("score")
+
+            factor_row = factor_row + "{0}: {1}\n".format(factor_name, factor_score)
+
+        f["Factors"] = factor_row
+        factor_scores.append(f)
+
+    demisto.debug("factor_scores: {0}".format(factor_scores))
+
+    markdown = tableToMarkdown("Historical Factor Scores for Domain [`{0}`](https://{0})".format(domain), factor_scores)
+
+    results = CommandResults(
+        readable_output=markdown,
+        outputs_prefix="SecurityScorecard.Company.FactorHistory",
+        outputs=entries
+    )
+
+    return results
+
+
+def securityscorecard_alert_grade_change_create_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+
+    """securityscorecard_alert_grade_change_create_command: Create alert based on grade
+
+    See https://securityscorecard.readme.io/reference#post_users-by-username-username-alerts-grade
+
+    :type ``client``: ``Client``
+    :type `` args``: ``Dict[str, Any]``
+        ``args['change_direction']``: Direction of change. Possible values are 'rises' or 'drops'., type ``String``
+        ``args['score_types']``: Types of risk factors to monitor.
+        Possible values are:
+        'overall',
+        'any_factor_score',
+        'network_security',
+        'dns_health',
+        'patching_cadence',
+        'endpoint_security',
+        'ip_reputation',
+        'application_security',
+        'cubit_score',
+        'hacker_chatter',
+        'leaked_information',
+        'social_engineering'. For multiple factors, ['leaked_information', 'social_engineering'], type ``array``
+        `args['target']``: What do you want to monitor with this alert.
+        It could be one of the following 'my_scorecard', 'any_followed_company', or an array of portfolio IDs,
+        e.g. xxxxxxxxxxxxxxxxxxxxxx,yyyyyyyyyyyyyyyyyyyyy, type ``array``
+
+    :return:
+        A ``CommandResults`` object that is passed to ``return_results``
+    :rtype: ``CommandResults``
+    """
+
+    email = demisto.params().get('username').get("identifier")
+    change_direction = str(args.get('change_direction'))
+    score_types = argToList(args.get('score_types'))
+    target = argToList(args.get('target'))
```

Comment:
According to their API docs:

>`target` (array of strings)
> what do you want to monitor with this alert setting, it could be one of the following values: "my_scorecard", "any_followed_company" or portfolio ids

So they're expecting a list in this case even though they're mentioning that they will accept either `my_scorecard` OR `any_followed_company` if we were to not specify portfolio IDs.

In the Integration, we provide 2 different parameters: `portfolios` and `target` where target is expected to be one value and portfolios can be multiple. Therefore `portfolios` has `isArray:true` while `target` doesn't.

In the code what I did was:
```python
target = target_arg or portfolios

...

response = client.create_grade_change_alert(
        ...
        target=argToList(target)
)

response = client.create_score_threshold_alert(
        ...
        target=argToList(target)
)